package.json

```diff
@@ -130,16 +128,14 @@
     "gettext-extractor": "^3.7.2",
     "gettext-parser": "^7.0.0",
     "glob": "^10.0.0",
-    "jest": "^29.0.1",
-    "jest-environment-jsdom": "29.7.0",
-    "jest-serializer-vue": "^3.1.0",
-    "jest-transform-stub": "^2.0.0",
+    "jsdom": "^22.1.0",
```

Comment:
Just as a comment:
if it works than ok, but I recommend using happy-dom instead. As jsdom does not provide any new API like buffer etc (unrelated to vitest vs jest but got errors where some API was missing with jsdom).

Reply:
I didn't try with `happy-dom`, but I can do that. I just took `jsdom`, because you did in #3565 😉

Reply:
All good, just wanted to mention that.

Reply:
I switched to `happy-dom` now and it seems to run slightly faster.